### DISCUSSION TITLE
How can I change the template in an existing mindmap?

### DISCUSSION THREAD

**Discussion by M-ax-G** [freeplane/freeplane#1962] [Help] [answered]
The title says it: How can I change the template in an existing mindmap?

I tried all option in Format | Manage styles, to no avail.

I would be happy to change the .mm file in case a simple name change of a property will do the trick?!

**Comment by fnatter** [comment, marked as answer]
hello M-ax-G,

the functionality is in Format->Manage Styles:

Format->Manage Styles->Copy map styles from... 
OR
Format->Manage Styles->Replace map styles by... 

Standard templates -> select template

[ ] Follow style changes

**Reply by M-ax-G** [reply]
Thanks! Worked like a charm. Not sure if/why I overlooked this command in the list.
